[DOCS-1956] Update docs for hashed API keys (#2018)
wandb/docs: support-tags.mdx

For comprehensive information about service accounts, including best practices and detailed setup instructions, refer to [Use service accounts to automate workflows](/platform/hosting/iam/service-accounts). For information about how service accounts behave in team contexts, refer to [Team Service Account Behavior](/platform/app/settings-page/teams#team-service-account-behavior).

<TeamServiceAccountCreate/>

> The full API key is only shown once at creation time. After you close the dialog, you will not be able to view it again.

Reviewer: ```suggestion
The full API key is only shown once at creation time. After you close the dialog, you cannot view it again.
```

Author: Done (Github didn't make this easy to apply for some reeason)
